rc/extra/gdb.kak

```diff
@@ -174,7 +174,7 @@ define-command gdb-session-stop %{
                 rm "${kak_opt_gdb_dir}/pid" "${kak_opt_gdb_dir}/pipe"
                 rmdir "$kak_opt_gdb_dir"
             else
-                printf %s\\n raise
+                echo raise
```

Comment:
I think that since the code after this scope is fully static, you could just print it with an `echo` in the other conditional branch and not rely on this trick (maybe one day there will be a valid `raise` keyword`, and this code will be broken).

```
if [ -n "$kak_opt_gdb_dir" ]; then
  kill $(ps -o pid= --ppid $(cat "${kak_opt_gdb_dir}/pid"))
  rm "${kak_opt_gdb_dir}/pid" "${kak_opt_gdb_dir}/pipe"
  rmdir "$kak_opt_gdb_dir"

  echo '
         set-option global gdb_dir ""

         gdb-clear-location
         gdb-clear-breakpoints

         remove-hooks global gdb
  '
fi
```

Unless you rely on an exception being raised here, in the calling scope of the command?